plugins/modules/ec2_vpc_endpoint_service_info.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: ec2_vpc_endpoint_service_info
+short_description: retrieves AWS VPC endpoint service details
+version_added: 1.5.0
+description:
+  - Gets details related to AWS VPC Endpoint Services.
+requirements: [ boto3 ]
```

Comment:
```suggestion
```

post #298 this isn't needed in amazon.aws (https://github.com/ansible-collections/community.aws/pull/569 is still pending)

Reply:
Shoot, 569 is totally on me - thanks!  :)

Reply:
569 has now been merged, not sure how you'd like to handle the updates here.